server/src/shared/app/base-controller.ts

```diff
@@ -1,6 +1,6 @@
 import * as express from 'express'
 
-export abstract class BaseController {
+export default abstract class BaseController {
```

Comment:
```suggestion
export abstract class BaseController {
```

Let's stick to regular exports; [apparently default exports are bad](https://basarat.gitbook.io/typescript/main-1/defaultisbad).